Ex1/5.3.whisperer.py

```diff
@@ -0,0 +1,35 @@
+# const vars
+SIZE = 5
+PATH = 'resources/logo.jpg'
+
+
+def print_secret_messages(path):
```

Comment:
https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing. 'Path' is a bad variable name.